Library/Formula/cucumber-cpp.rb

```diff
@@ -0,0 +1,67 @@
+class CucumberCpp < Formula
+  desc "Support for writing Cucumber step definitions in C++"
+  homepage "https://cucumber.io"
+  url "https://github.com/cucumber/cucumber-cpp/archive/v0.3.tar.gz"
+  sha256 "1c0f9949627e7528017bf00cbe49693ba9cbc3e11087f70aa33b21df93f341d6"
+
+  depends_on "cmake" => :build
+  depends_on "boost"
+
+  def install
+    args = std_cmake_args
+    args << "-DCUKE_DISABLE_GTEST=on"
+    args << "-DCUKE_DISABLE_CPPSPEC=on"
+    args << "-DCUKE_DISABLE_FUNCTIONAL=on"
+    args << "-DCUKE_DISABLE_BOOST_TEST=on"
+    system "cmake", ".", *std_cmake_args
+    system "cmake", "--build", "."
+    include.install "include/cucumber-cpp"
+    lib.install Dir["src/*.a"]
+  end
+
+  test do
+    ENV["GEM_HOME"] = testpath
+    ENV["BUNDLE_PATH"] = testpath
+    ENV.prepend_path "PATH", testpath/"bin"
+    system "gem", "install", "cucumber"
+
+    (testpath/"features/test.feature").write <<-EOS.undent
+      Feature: Test
+        Scenario: Just for test
+          Given A given statement
+          When A when statement
+          Then A then statement
+    EOS
+    (testpath/"features/step_definitions/cucumber.wire").write <<-EOS.undent
+      host: localhost
+      port: 3902
+    EOS
+    (testpath/"test.cpp").write <<-EOS.undent
+      #include <cucumber-cpp/defs.hpp>
+      GIVEN("^A given statement$") {
+      }
+      WHEN("^A when statement$") {
+      }
+      THEN("^A then statement$") {
+      }
+    EOS
+    system ENV.cxx, "test.cpp", "-L#{lib}", "-lcucumber-cpp", "-o", "test",
+      "-lboost_regex", "-lboost_system"
+    begin
+      pid = fork { exec "./test" }
+      assert_match %r{^Feature: Test
```

Comment:
The only problem is that there is a last line written by cucumber telling the execution time that might not be constant. That's why I wanted to use a regex.